EmployeePortal using SpringMVC(JAVA config based using JDBC)/src/main/java/com/employeeportal/controllers/EmployeeController.java

```diff
@@ -0,0 +1,55 @@
+package com.employeeportal.controllers;
+
+import com.employeeportal.model.Employee;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.ModelAndView;
+import com.employeeportal.service.EmployeeServiceImpl;
+import java.util.List;
+
+@Controller
+public class EmployeeController {
+
+    @Autowired
+    private EmployeeServiceImpl employeeService;
+
+    @RequestMapping(value = "/empform")
+    public ModelAndView showForm() throws Exception {
+        return new ModelAndView("/empform","command",new Employee());
+    }
+
+    @RequestMapping(value="/save",method = RequestMethod.POST)
+    public ModelAndView save(@ModelAttribute("employee") Employee employee) throws Exception {
+        employeeService.save(employee);
+        return new ModelAndView("redirect:/viewemp");
+    }
+
+    @RequestMapping(value = "/viewemp",method = RequestMethod.GET)
+    public ModelAndView viewemp() throws Exception {
+        List<Employee> list = employeeService.getEmployees();
+        ModelAndView modelAndView = new ModelAndView();
+        modelAndView.addObject("list",list);
+        modelAndView.setViewName("viewemp");
+        return modelAndView;
+    }
+
+    @RequestMapping(value="/editemp/{id}")
```

Comment:
You can use "/employee/edit{id}" as a request mapping, however "/employee" can be extracted outside the class.